mentor.html

```diff
@@ -3,40 +3,150 @@
+    <div class="row">
+      <div class="col-md-8 col-md-offset-2">
         <div class="clearfix"></div>
-        <h2 class="section-heading">What makes a good mentor?</h2>
+        <h3>1. Mentees First</h3>
+        <p>
+          We believe in putting the mentees needs first. You need to emphasize
+          with the mentee, understand his personality, requirements and
+          circumstances. Make an effort to understand where the mentee comes from
+          and what you need to do, to get them to contribute to free software.
+          Mentoring is about them, not about your or openSUSEs needs.
+        </p>
+        <h3>2. Build a Relationship and have a lot of fun...</h3>
+        <p>
+          The most important goal is that you build a relationship based on trust
+          and respect with the mentee and that you have fun together. Without this
+          everything else you'll do is in vein! Such relationships do not just
+          happen, they need ongoing effort and monitoring, especially during the
+          early stages.
+        </p>
+        <h3>3. Be a role model</h3>
+        <p>
+          Today this sounds like a weird thing from the 60s and it makes most
+          people uncomfortable but you need to be a role model for your mentee!
+          Free Software development is as much about behavior in a community,
+          knowing what you want and the ability to convey this to others than it
+          is about pointers, gcc macros and git command line options. You need
+          to ensure that your mentee can follow your example and that you're
+          not passing on your bad habits. You are responsible for developing
+          the competence AND character of the mentee!
+        </p>
+        <h3>4. Plan</h3>
+        <p>
+          The first thing you'll do together with the mentee is to prepare for the
+          far future. Precise, reasonable and well-thought-out planing is a must
+          for every successful mentoring project. Getting into free software
+          development can be an overwhelming and daunting endeavor. A decent
+          plan provides a path and helps to take one step after the other into
+          this strange new world. It's your duty that the plan the mentee comes
+          up with, is do-able, provides orientation and is technically sound.
+          You're the expert!
+        </p>
+        <h3>5. Track Progress</h3>
+        <p>
+          We encourage you to set up regular milestones in your plan that get
+          reviewed and celebrated by you and your mentee. It's important to
+          understand where you are and how you progress. Additionally we believe
+          that meeting as often as you can will help, over time, to build the
+          relationship and bonding with your mentee. The consistency of your
+          relationship is very important. Find a regular schedule where you meet
+          or even better meet as often as you can!
```

Comment:
I would actually explicitly recommend daily meetings. "Meet as often as you can" sounds a bit overwhelming. Maybe say something like "Do short daily meetings to check in on progress and address obstacles. Try to spend some time on pair programming to get a feel for how things are moving and produce some quality code."